.gitmodules

```diff
@@ -3,4 +3,5 @@
 	url = https://github.com/open-source-parsers/jsoncpp.git
 [submodule "tools/rpc_spec"]
 	path = tools/rpc_spec
-	url = https://github.com/smartdevicelink/rpc_spec.git
```

Comment:
this shall be removed before merge.

Reply:
@yang1070 
Thank you for leaving a comment.
Please approve these changes.